examples/tutorials/advance_model_config/03_train_steps.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>which <strong>is</strong> called embedding</p><p> </p><p> </p><p>Do you really need to explain in this tutorial what Encoder-Decoder architecture is?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>